tools/api/model.py

```diff
@@ -0,0 +1,65 @@
+import sys
+
+sys.path.insert(1, '../validators/ontology_validator/')
```

Comment:
This should not exist in a library file.  Maybe you need a setup.py in tools/explorer?